### PR TITLE
Add instructions to monitor Fleetspeak servers

### PR DESCRIPTION
**Pull request by tsehori**
Once google/grr#796 is merged, GRR will contain a sample [Grafana dashboard](https://grafana.com/docs/grafana/latest/features/dashboard/dashboards/#dashboard-overview) for [Fleetspeak](https://github.com/google/fleetspeak/) servers alongside the [existing sample dashboards](https://github.com/google/grr/tree/master/monitoring/grafana) for GRR server components.

The motivation of this PR is to teach our users how to configure Fleetspeak server and Prometheus correctly so that Fleetspeak can be scraped by [Prometheus](https://prometheus.io/) and integrated with Grafana, which will allow our users to use the provided sample dashboard for Fleetspeak server.

### CODE REVIEW

**Comment by tsehori** [comment]
I also fixed a small error in the instructions for installing Grafana.

File: maintaining-and-tuning/monitoring.md [hunk 1]
```diff
@@ -155,38 +172,42 @@ access to the port, for example by employing a firewall. Furthermore, read
 [Prometheus Security](https://prometheus.io/docs/operating/security/).
 
 ## Example visualization and alerting setup
-This example will walk you through setting up Grafana as a dedicated visualization software to parse, display and query metrics scraped from GRR server components by Prometheus. You will also be able to set up a simple alerting system using Grafana.
-These instructions assume that GRR server and Prometheus are both up and running.
+This example will walk you through setting up Grafana as a dedicated visualization software to parse, display and query metrics scraped from GRR server components by Prometheus. If Prometheus was configured to scrape Fleetspeak servers as well, all steps apply. You will also be able to set up a simple alerting system using Grafana.
```
Comment (mbushkov) [inline maintaining-and-tuning/monitoring.md:175]: >  If Prometheus was configured to scrape Fleetspeak servers as well, all steps apply. 

And if it wasn't? It's not clear from this sentence, which steps do/don't apply then.

File: maintaining-and-tuning/monitoring.md [hunk 2]
```diff
@@ -155,38 +172,42 @@ access to the port, for example by employing a firewall. Furthermore, read
...
+1. You can now use the dashboards. The dashboards can give a general overview over the main components of the GRR server anf Fleetspeak servers, which can be utilized by the user to monitor different metrics of each component. Examples for such metrics can be found in the [examples above](#example-queries). Remember that the dashboards and alerts are flexible, and can be expanded or modified to adjust to your exact needs. Additional metrics can be used by exploring `http://<host>:<port>/metrics` for each component of GRR server (change the port according to the GRR server component you want) and Fleetspeak servers, and if you wish to create your own [custom dashboards](https://grafana.com/docs/grafana/latest/getting-started/getting-started/#create-a-dashboard), [panels](https://grafana.com/docs/grafana/latest/panels/add-a-panel/#add-a-panel) and [alerts](https://grafana.com/docs/grafana/latest/alerting/create-alerts/#create-alerts), make sure to go over the corresponding documentation in Grafana.
```
Comment (mbushkov) [inline maintaining-and-tuning/monitoring.md:213]: Typo: anf -> and

Comment (mbushkov) [reply maintaining-and-tuning/monitoring.md:213]: While we're at it, I'd probably replace "if you wish" with "if you'd like". It sounds simpler and I think I rarely saw "wish" in tech documentation.

File: maintaining-and-tuning/monitoring.md (hunk 1 above)
Comment (tsehori) [reply maintaining-and-tuning/monitoring.md:175]: You're right; Prometheus would have been configured to scrape Fleetspeak back in the example setup from the top of the doc. I'll link it for clarity.

**Comment by tsehori** [comment]
ptal

**Pull request merged**